VimeoNetworking/Sources/Models/VIMUser.m

```diff
@@ -152,7 +157,14 @@ - (void)parseConnections
             NSDictionary *value = [dict valueForKey:key];
             if([value isKindOfClass:[NSDictionary class]])
             {
-                VIMConnection *connection = [[VIMConnection alloc] initWithKeyValueDictionary:value];
+                Class connectionClass = [key isEqualToString:VIMConnectionNameNotifications] ? [VIMNotificationsConnection class] : [VIMConnection class];
+                VIMConnection *connection = [[connectionClass alloc] initWithKeyValueDictionary:value];
```

Comment:
Do you know why these aren't already mapped by the object mapper? Why do we have to parseConnections manually?